yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -315,26 +316,38 @@ def make_args(file, args, name, number):
             args += self._configuration_args(self.basename, keys)
             if name == 'i':
                 args.append('-i')
-            return (
-                [encodeArgument(arg) for arg in args]
-                + [encodeFilename(self._ffmpeg_filename_argument(file), True)])
+            return [encodeArgument(arg) for arg in args] + [encodeFilename(self._ffmpeg_filename_argument(file), True)]
 
         for arg_type, path_opts in (('i', input_path_opts), ('o', output_path_opts)):
-            cmd += itertools.chain.from_iterable(
-                make_args(path, list(opts), arg_type, i + 1)
-                for i, (path, opts) in enumerate(path_opts) if path)
+            for i, (path, opts) in enumerate(path_opts):
+                if path:
+                    cmd += make_args(path, list(opts), arg_type, i + 1)
```

Comment:
why?

Reply:
I felt like the original code was overly complicated with no good reason. This expression is way easier to read, takes the same amout of lines and doesn't increase runtime

Reply:
I disagree. The current implementation is fine. Besides, this is not a re-factoring PR. Do not make unrelated changes